extension/src/providers/gotoProvider.ts

```diff
@@ -84,9 +84,9 @@ export class AutolispDefinitionProvider implements vscode.DefinitionProvider{
 			}
 		} while (flag);
 		// If we still haven't found anything check the 1st occurrence of setq's. This will find globals setqs and nested ones possibly inside other defuns
-		if (result.length === 0){
```

Comment:
try this codes, and it will not break in the above loop:
```
(defun test (num) 
  (print num)
  (print "I am in the same line")
  (print "hello")
  (inner (* 20 5))
)
(defun inner (var) 
  (print var) ;; try F12 on this var
  (print "world")
)
(setq str (test 20)) 
```

Reply:
Yeah, I am currently researching that. I am thinking I shouldn't have fixed the Sexpression, I should have checked the variable for null and acted accordingly.

Reply:
It looks like the changes in Sexpression is to fix the edge case for (defun test ()), not for the variables